source/refunding_payments/index.html.md.erb

```diff
@@ -95,15 +97,15 @@ If GOV.UK Pay could not successfully create the refund, the JSON response will c
 
 ## Checking the status of a refund
 
-`GET /v1/refunds/PAYMENT-ID`
+`GET /v1/payments/PAYMENT-ID/refunds/REFUND-ID`
```

Comment:
How was this wrong for so long? 😬

Reply:
Good question! Did the endpoint change at some point maybe?